docs/attributes-registry/profile.md

```diff
@@ -0,0 +1,51 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# Profile
+
+- [Profile Build Id](#profile-build-id-attributes)
+- [Profile Frame](#profile-frame-attributes)
+- [Profile Link](#profile-link-attributes)
+
+## Profile Build Id Attributes
+
+Unique identifier for an executable.
+
+| Attribute              | Type   | Description                                                               | Examples                                                                              | Stability                                                        |
+| ---------------------- | ------ | ------------------------------------------------------------------------- | ------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `profile.build_id.gnu` | string | The GNU build ID as can be found in the `.note.gnu.build-id` ELF section. | `c89b11207f6479603b0d49bf291c092c2b719293`                                            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
is there some documentation on this? could it be referenced? So someone not familiar with profiling could go and learn what it means?

Reply:
Would it help to link here the GNU build system article from wikipedia?  https://wikipedia.org/wiki/GNU_build_system

Reply:
We typically don't use signal-specific prefixes for attribute names, unless the attribute is only meaningful just for that one signal. Is `build_id` strictly a profiling concept? If it is an identifier of a build I think it may possibly be used in other signals (e.g. log an event and include information about the build of the executable that logged it). In that case we should not use `profile.` namespace and should put this attribute in an appropriate top-level namespace (or create one if none exists).
The same comment applies to other attributes introduced.

Reply:
@tigrannajaryan I do need some advice on resolving a conflict.

With `process.executable` there exists an attribute group, where `process.executable.build_id` could be another field. But for the Profiling use case, there is a conflict, that a single process can have multiple executables - e.g. a stack trace of a Python script can have libc, libpython, kernel and interpreted python frames (and maybe even more). The former three, libc, libpython and kernel, will have their dedicated `build_id`. So a 1:1 mapping of process to executable is not possible.
How could the 1:1 mapping become a 1:n mapping where one process can have multiple executables in its stack?  

FYI: @christos68k @AlexanderWert

Reply:
Removed the build_id from this proposed change with https://github.com/open-telemetry/semantic-conventions/pull/1188/commits/2ae1ac9c7156438f3698e9b0fb1f5ca182157926 and will open a new issue/PR around it as a follow up.